### DISCUSSION TITLE
I cannot update my email settings (timeout request), where can I update them on the server?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2211] [Help]
Hello,

I cannot set my email settings...
I got a timeout.
Where can I update them on the server (it is not the .env file)?

Thank you

**Comment by mjashanks** [comment]
I can reproduce this, I have logged this here: https://github.com/Budibase/budibase/issues/2215.

I tried to figure out how to set it via the DB, but couldn't work it out - will try more and get back to you.

**Comment by oswaldB** [comment]
Thank you

**Reply by mjashanks** [reply]
I just got mine to work by selecting

![image](https://user-images.githubusercontent.com/3524181/127873689-fd97c516-b375-4902-8b5c-f01fae8c5436.png)

I.e. my settings were wrong.

If you look at the eventual response in from `/api/admin/configs`, do you get `upstream request timeout`? This would suggest that the request to your provider is timing out.
